### ISSUE TITLE
Flaky ClientReconnectTest

### ISSUE THREAD

**Issue by puzpuzpuz** [hazelcast/hazelcast-nodejs-client#658]
See http://jenkins.hazelcast.com/job/NodeJS-pr-builder/663/testReport/(root)/%22after%20each%22%20hook%20for%20%22create%20proxy%20while%20member%20is%20down,%20member%20comes%20back%22/ClientReconnectTest__after_each__hook_for__create_proxy_while_member_is_down__member_comes_back_/

```
Error Message
Listener cannot be added!
Stacktrace
Error: Listener cannot be added!
    at client.getInvocationService.invokeUrgent.then.then.catch.err (lib/listener/ListenerService.js:21:68)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Comment by puzpuzpuz** [comment]
Closing this one as it seems to be no longer actual. Will reopen if it shows itself in the future.

**Comment by srknzl** [comment]
Fails on gh actions reopening https://github.com/hazelcast/client-compatibility-test/runs/2912680402?check_suite_focus=true

**Comment by srknzl** [comment]
Closed since it fails only below 4.1. Maybe we can add a version check for backward compatibility tests.
